### DISCUSSION TITLE
Mi az eljárás, ha nem értünk egyet a bevfeldata XML tartalmával?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#43] [Kérdések és válaszok - Gépi interfész]
Kedves NAV Szakértők!

Megértésünk szerint a manageDeclarationSubmissionResponse-ban érkezik meg a NAV által összeállított bevfelddata XML file. Ezzel kapcsolatosan lenne még két kérdésünk:

1. Ha nem értünk egyet, akkor kell-e jelezni, hogy nem hagytuk jóvá?
2. Lehetséges-e ekkor a tényleges bevallást az ÁNYK-n keresztül beküldeni az adott időszakra?

Nagyon köszönjük előre is!

**Comment by renced42** [comment]
Kedves @szalaierika 

A /manageDeclarationSubmission az áfabevallás benyújtására szolgáló operáció, ott nem érkezik az ügyfélnek adat.
Az eÁfa bevallásban letölthető bevallás XML az ügyféli döntéseket tükrözi azt már az ügyfél által megadott adatok alapján generálja a rendszer. Ha saját magával kapcsolatosan az ügyfél nem ért egyet :) , akkor önellenőrzést kell majd beadnia az általa beadott bevallásra.
